share/pot/start.sh

```diff
@@ -152,7 +152,6 @@ _js_vnet()
 	_ip=$( _get_ip_var "$_pname" )
 	## if norcscript - write a ad-hoc one
 	if [ "$(_get_conf_var "$_pname" "pot.attr.no-rc-script")" = "YES" ]; then
-		touch "${POT_FS_ROOT}/jails/$_pname/m/tmp/tinirc"
 		cat >>"${POT_FS_ROOT}/jails/$_pname/m/tmp/tinirc" <<-EOT
```

Comment:
removed these calls to touch, assuming they're not really needed (added address selection policy will always have created the file at this point). Please let me know if they should be put back in.

Reply:
I guess `touch` is superfluous, `cat` will generate the file anyway